tests/documentation/command-reference/doc_command_reference_deletenamespace_test.go

```diff
@@ -0,0 +1,75 @@
+package docautomation
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/google/go-cmp/cmp"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-developer/odo/tests/helper"
+)
+
+var _ = Describe("doc command reference odo delete namespace", func() {
+	var commonVar helper.CommonVar
+	var commonPath = filepath.Join("command-reference", "docs-mdx", "delete-namespace")
+	var outputStringFormat = "```console\n$ odo %s\n%s```\n"
+
+	BeforeEach(func() {
+		commonVar = helper.CommonBeforeEach()
+		helper.Chdir(commonVar.Context)
+		Expect(helper.VerifyFileExists(".odo/env/env.yaml")).To(BeFalse())
+	})
+
+	AfterEach(func() {
+		helper.CommonAfterEach(commonVar)
+	})
+
+	Context("To delete a namespace resource", func() {
+
+		BeforeEach(func() {
+			helper.Cmd("odo", "create", "namespace", "odo-dev").ShouldRun()
+
+		})
+
+		AfterEach(func() {
+			commonVar.CliRunner.DeleteNamespaceProject("odo-dev", false)
```

Comment:
You will need to pass true here, or the next test trying to create the same namespace will fail (because this deletion is not terminated)

Reply:
Ok, got it...Thank you!